packages/wonder-blocks-switch/src/themes/default.ts

```diff
@@ -0,0 +1,66 @@
+import {tokens} from "@khanacademy/wonder-blocks-theming";
+
+const theme = {
+    color: {
+        bg: {
+            primary: {
+                off: tokens.color.offBlack50,
+                disabledOff: tokens.color.offBlack32,
+                activeOff: tokens.color.offBlack64,
+                on: tokens.color.blue,
+                disabledOn: tokens.color.fadedBlue,
+                activeOn: tokens.color.activeBlue,
+            },
+            secondary: {
+                on: tokens.color.white,
+                off: tokens.color.white,
+            },
+            tertiary: {
+                on: tokens.color.blue,
+                disabledOn: tokens.color.fadedBlue,
+                off: tokens.color.offBlack50,
+                disabledOff: tokens.color.offBlack32,
+            },
+        },
+        outline: {
+            default: tokens.color.blue,
+        },
+    },
+    border: {
+        radius: {
+            // slider
+            small: tokens.spacing.small_12,
+            // switch
+            full: tokens.border.radius.full,
+        },
+    },
+    size: {
+        height: {
+            none: 0,
+            // switch
+            medium: 20,
+            // slider
+            large: tokens.spacing.large_24,
+        },
+        width: {
+            none: 0,
+            xxxxSmall: tokens.spacing.xxxxSmall_2,
+            // NOTE: This token is specific to the Switch component
+            // switch
+            medium: 20,
+            // NOTE: This token is specific to the Switch component
+            // slider
+            xLarge: 40,
+        },
+        offset: {
+            default: 1,
+        },
+    },
+    spacing: {
+        xxxxSmall: tokens.spacing.xxxxSmall_2,
+        xxxSmall: tokens.spacing.xxxSmall_4,
+        medium: tokens.spacing.medium_16,
```

Comment:
**suggestion:** Similar suggestion here around naming. In fact, by looking at the implementation, I could see another pattern emerging:

https://github.com/Khan/wonder-blocks/pull/2029/files#diff-f0d673a56005e5635e2d3ead871ffdb309a4166f246fedfd77e63a396865c896R159-R160

How do you feel about using a different naming convention in here (similar to `surface` as mentioned in a previous comment):

- `xxxxSmall` => `sliderPosition` or `slider.position` (as it is used for `top` and `left`).
- `xxxSmall` => `iconPosition` or `icon.position`.
- `medium` => `switchTransform` or `switch.transform`. (+ the whole transform value could be included in here).

Of course, this are some thoughts I'm having while I'm looking at the code, so happy to chat more offline about this! I'm even checking about if/how this can be integrated with the `button` theme.